package.json

```diff
@@ -15,7 +15,9 @@
     "underscore": "1.12.1",
     "hosted-git-info": "3.0.8",
     "url-parse": "1.5.1",
-    "postcss": "8.2.10"
+    "postcss": "8.2.10",
+    "browserslist": "4.16.6",
+    "dns-packet": "5.2.2"
```

Comment:
@mukama I just noticed the version of dns-packet is a jump from 1 to 5. That's seems a bit risky. I think we need to fix and consider these dependencies in a separate PR. Rebecca and I are going to look into this.

Reply:
I completely agree @jroebu14. For now let me get rid of these security patches so that they don't conflict with you and Rebecca's fixes

Reply:
Doing this will however break the pipeline for this PR until fixes are merged